lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -1010,6 +1010,11 @@ void HdVP2Mesh::Sync(
             for (size_t i = 0; i < numFaceVertexIndices; i++) {
                 const int sceneFaceVtxId = faceVertexIndices[i];
 
```

Comment:
See line 1005, which created the buffer filled with -1, assuming the number of vertices would reflect the highest possible index. Sometimes, those indices can be greater, probably when geometry is processed and some vertices are dropped?

Whatever is the cause, we need to validate the input data when creating the rapid lookup tables.